flink-connectors/flink-connector-cassandra/src/test/java/org/apache/flink/batch/connectors/cassandra/CassandraOutputFormatBaseTest.java

```diff
@@ -202,9 +202,12 @@ public void testReleaseOnFailure() throws Exception {
 
             assertThat(testCassandraOutputFormat.getAvailablePermits()).isEqualTo(1);
             assertThat(testCassandraOutputFormat.getAcquiredPermits()).isEqualTo(0);
+        try {
+            testCassandraOutputFormat.close();
         } catch (IOException ignored) {
```

Comment:
use assertThatThrownBy

Reply:
I did that because the aim of the test is to assert on semaphore count and not on exception as I wrote in the comment in the code. But if you think it is not misleading to assert on the exception, then fine.

Reply:
done